recipes/android-ndk/all/conanfile.py

```diff
@@ -1,10 +1,11 @@
-from conan import ConanFile, tools
+from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, download, unzip, load, copy
 import os
 import re
 import shutil
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.43.0"
```

Comment:
```suggestion
required_conan_version = ">=1.46.0"
```

Due to copy and unzip

Reply:
This is required @SpaceIm?

Reply:
Yes, due some imports.